x-pack/plugins/security_solution/server/endpoint/services/actions/clients/endpoint_actions_client.ts

```diff
@@ -84,7 +84,7 @@ export class EndpointActionsClient extends ResponseActionsClientImpl {
       await updateCases({
         casesClient: this.options.casesClient,
         endpointData: agentIds.hosts,
-        createActionPayload: createPayload,
+        createActionPayload: { ...createPayload, action_id: response.id },
```

Comment:
Does this change also need to be made here: https://github.com/elastic/kibana/blob/5fa14686e5c6b3edffb338c3029a46a4316fdfca/x-pack/plugins/security_solution/server/endpoint/services/actions/clients/lib/base_response_actions_client.ts#L110-L122

This `updateCases()` function is not called for S1 host and will also be deleted once I completely refactor the Endpoint client to use the methods from the base class.

Reply:
Good catch, I didn't see the updateCases function lives here, I assumed that update_cases.ts is the only one we use. I'll take a closer look at this.